specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -29,12 +29,15 @@ enum ImageCategory {
 enum AnalyzeTextOutputType {
   @doc("Output severities in four levels, the value could be 0, 2, 4, 6.")
   FourSeverityLevels,
+
+  @doc("Output severities in eight levels, the value could be 0,1,2,3,4,5,6,7.")
+  EightSeverityLevels,
 }
 
 @doc("The type of image analysis output.")
 enum AnalyzeImageOutputType {
   @doc("Output severities in four levels, the value could be 0, 2, 4, 6.")
-  FourSeverityLevels,
+  FourSeverityLevels
```

Comment:
Nit: terminal commas are preferred, actually. Makes subsequent diffs smaller and easier to review.

Reply:
updated